node/src/components/gossiper/gossip_table.rs

```diff
@@ -598,25 +529,11 @@ mod tests {
         assert_eq!(GossipAction::AwaitingRemainder, action);
         check_holders(&node_ids[..2], &gossip_table, &data_id);
 
-        // Pause gossiping and check same partial data from third source causes `Noop` to be
-        // returned and holders updated.
-        gossip_table.pause(&data_id);
-        let action = gossip_table.new_partial_data(&data_id, node_ids[2]);
-        assert_eq!(GossipAction::Noop, action);
-        check_holders(&node_ids[..3], &gossip_table, &data_id);
-
-        // Reset the data and check same partial data from fourth source causes `AwaitingRemainder`
-        // to be returned and holders updated.
-        gossip_table.resume(&data_id).unwrap();
-        let action = gossip_table.new_partial_data(&data_id, node_ids[3]);
-        assert_eq!(GossipAction::AwaitingRemainder, action);
-        check_holders(&node_ids[..4], &gossip_table, &data_id);
-
         // Finish the gossip by reporting three infections, then check same partial data causes
         // `Noop` to be returned and holders cleared.
         let _ = gossip_table.new_complete_data(&data_id, Some(node_ids[0]));
         let limit = 4 + EXPECTED_DEFAULT_INFECTION_TARGET;
-        for node_id in &node_ids[4..limit] {
+        for node_id in &node_ids[3..limit] {
```

Comment:
Why do we need to infect one more now?

Reply:
We shouldn't - I'll update it and the other similar instance in the `new_complete_data()` test.

Reply:
Fixed in [a3a385a](https://github.com/casper-network/casper-node/pull/1475/commits/a3a385ab94e1ee3fc31df04a17e9e5fdfda8f86f).